src/Symfony/Bundle/FrameworkBundle/Controller/RedirectController.php

```diff
@@ -146,4 +146,13 @@ public function urlRedirectAction(Request $request, string $path, bool $permanen
 
         return new RedirectResponse($url, $statusCode);
     }
+
+    public function __invoke(Request $request): Response
+    {
+        if (null !== $route = $request->attributes->get('route')) {
```

Comment:
That looks like alot of magic. It might be cleaner to just create a new controller with invoke for one action. And possibly deprecate it in RedirectController so there is no ambiguity.